ballerina/subscriber_client.bal

```diff
@@ -97,13 +97,13 @@ isolated function buildSubscriptionChangeRequest(string mode,
 
 isolated function processHubResponse(string hub, string mode, 
                                      SubscriptionChangeRequest subscriptionChangeRequest,
-                                     http:Response|http:PayloadType|error response) returns SubscriptionChangeResponse|SubscriptionInitiationError {
+                                     http:Response|error response) returns SubscriptionChangeResponse|SubscriptionInitiationError {
 
     string topic = subscriptionChangeRequest.topic;
     if response is error {
         return error SubscriptionInitiationError("Error occurred for request: Mode[" + mode+ "] at Hub[" + hub + "] - " + response.message());
     } else {
```

Comment:
Remove the else scope as well

Reply:
Fixed with [24c3f49387a532b745fb6bd41fe18bebb7e48f29](https://github.com/ballerina-platform/module-ballerina-websub/pull/520/commits/24c3f49387a532b745fb6bd41fe18bebb7e48f29)